# latitude and longitude given in decimal degrees (* 10^7)
int32 latitude
int32 longitude

# height in mm
int32 altitude

# posAcc given in mm
uint16 posAcc

# max speed to travel to waypoint in % of 100
uint8 speedTo

# time to stay at waypoint in ms
uint16 holdTime

# yawFrom angle given in degrees (* 1000)
int32 yawFrom

# camera pan and tilt angles during waypoint travel given in degrees (* 1000)
int32 panAngle
int32 tiltAngle
